services/resource_dereference_service.go

```diff
@@ -32,7 +32,7 @@ func (rds ResourceService) DereferenceResourceMetadata(did string, resourceId st
 
 	var context string
 	if contentType == types.DIDJSONLD || contentType == types.JSONLD {
-		context = types.ResolutionSchemaJSONLD
+		context = types.W3IDDIDURL
```

Comment:
This change is unnecessary now as that URL is not live, it won't resolve to anything on Browser. Should be done when the URL/redirect is hosted

Reply:
Reverted these changes